grails-app/views/wps/show.gsp

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.1/css/bootstrap.min.css">
+        <link rel="stylesheet" href="${resource(dir: 'css', file: 'imosTheme.css')}" type="text/css">
+    </head>
+    <body>
+        <div class="imosHeader">
+            <div class="container">
+                <a class="btn" role="button" href="https://imos.aodn.org.au/imos123/home">
+                    <img src="https://static.emii.org.au/images/logo/IMOS-Ocean-Portal-logo.png" alt="IMOS logo">
+                </a>
+            </div>
+        </div>
+        <div class="container">
+            <h2><g:message code="job.status" default="WPS Status"/></h2>
+            <dl>
+                <g:labelledContent labelCode="job.id.label">${job.uuid}</g:labelledContent>
+                <g:labelledContent labelCode="job.createdTimestamp.label" if="${job.createdTimestamp}" >
+                  <joda:format value="${job.createdTimestamp}" />
+                </g:labelledContent>
+                <g:labelledContent labelCode="job.status.label">
+                  <g:message code="job.status.${job.status}" default="${job.status.toString()}" />
```

Comment:
I think if the job isn't completed it should say for example

**Status:** Submitted at 19/02/2015 3:30pm